project-base/storefront/components/Pages/Wishlist/Wishlist.tsx

```diff
@@ -31,59 +31,65 @@ export const Wishlist: FC = () => {
         );
     };
 
-    if (fetching) {
-        return <LoaderWithOverlay />;
-    }
-
     return (
         <>
-            <Heading className="!text-big lg:!text-h1" type="h1">
-                {t('Wishlist')}
+            <Heading type="h1">
+                {`${t('Wishlist')}${wishlist?.products.length ? ` (${wishlist.products.length})` : ''}`}
```

Comment:
Same with the title here. Not sure how much it would help but I feel like it would make it more clear.

Reply:
Sure. Refactored.